daemon/containerd/image_list.go

```diff
@@ -78,7 +78,7 @@ func (i *ImageService) Images(ctx context.Context, opts imagetypes.ListOptions)
 	}
 
 	// TODO(thaJeztah): do we need to take multiple snapshotters into account? See https://github.com/moby/moby/issues/45273
-	snapshotter := i.client.SnapshotService(i.snapshotter)
+	snapshotter := i.snapshotterService
```

Comment:
Hmm.. this means we won't be able to use multiple snapshotters (see the TODO comment above). If we do this, we should store it indexed by snapshotter.